uniswap_swap_bot/src/bot.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  Finding,
+  HandleTransaction,
+  TransactionEvent,
+  FindingSeverity,
+  FindingType,
+} from "forta-agent";
+
+import { UNISWAP_ADDRESS, SWAP_EVENT } from "./constant";
+
+const handleTransaction: HandleTransaction = async (
+  txEvent: TransactionEvent
+) => {
+  const findings: Finding[] = [];
+
+  const tokenSwapEvent = txEvent.filterLog(SWAP_EVENT, UNISWAP_ADDRESS);
+  tokenSwapEvent.forEach((swapEvent) => {
+    const { sender, recipient, amount0, amount1 } = swapEvent.args;
+
+    console.log("deposit and output", amount0, amount1);
+    findings.push(
+      Finding.fromObject({
```

Comment:
In the finding, it would be helpful to know _which_ token pair was swapped. So in addition to finding out when _any_ UniswapV3 pool has a `swap`, we would also like to see the specific trading pair that was swapped.